server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -73,6 +78,10 @@ object NettyConfig {
     shutdownEventLoopGroupOnClose = true,
     socketBacklog = 128,
     socketKeepAlive = true,
+    requestTimeout = 60.seconds,
```

Comment:
are there any "best practices" as to what the values here shuold be?

Reply:
I found a [website](https://www.tabnine.com/code/query/%22ChannelOption.CONNECT_TIMEOUT_MILLIS%22) containing Netty configuration examples, and I noticed several similar examples to ours. They show how to extract values from time-based options and map them to integers.

Reply:
I meant the specific values, no the way they are then parsed :)